.github/workflows/ci.yml

```diff
@@ -246,9 +246,28 @@ jobs:
           path: docs
           retention-days: 1
 
+  mkdocs:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Is there a way to also have documentation of the latest main branch and versions/x.0.0 branches so we can see what those look like? Although that might be confusing for users as they might not know what the latest version is then.

Reply:
I guess we could add another step, that uses the `mike` command generating the docs for the latest on versions/x.0.0 labeling it `next` (as the docker image). 

For main that would then mean we can call that `edge`. Though I am not sure we want to add those.

Reply:
Maybe we should start without those, and evaluate later if it is something we want?